packages/button/src/__demo__/examples/icons.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import Button from '../../Button';
+import { IconNewReleases } from '@mineral-ui/icon';
+
+const Root = createStyledComponent('div', {
+  '& > button': {
+    marginRight: '0.5rem'
+  }
+});
+
+export default {
+  title: 'With Icon',
+  description: 'Button with Icon.',
+  scope: { Button, IconNewReleases, Root },
+  source: `
+  () => {
```

Comment:
don't start the source string with a newline because then there will be a newline at the start of the editor